Documentation/tutorials/edge-bridge-tutorial.md

```diff
@@ -242,172 +254,220 @@ edgebridgetutorialapp://
+
+> [!TIP] 
+> If you want to map an array of objects in the source field to an array of different objects in the target field, add `[*]` after the array name in the destination field paths.
 
-| JSON Property                           | XDM Property                       | trackAction        | trackState         |
-| --------------------------------------- | ---------------------------------- | ------------------ | ------------------ |
-| data.contextdata.product\\.add\\.event  | commerce.productListAdds.value     | :white_check_mark: |                    |
-| data.contextdata.product\\.view\\.event | commerce.productListViews.value    |                    | :white_check_mark: |
-| data.contextdata.product\\.id           | productListItems.SKU               | :white_check_mark: | :white_check_mark: |
-| data.contextdata.product\\.name         | productListItems.name              | :white_check_mark: | :white_check_mark: |
-| data.contextdata.product\\.units        | productListItems.quantity          | :white_check_mark: |                    |
+<img src="../assets/edge-bridge-tutorial/data-prep/datastreams-mapping-target-property-array.png" alt="Datastream create calculated field" width="1100"/>  
 
-12. After completing all the mappings, click **Save**.
+After creating the desired mappings, select **Save** in the top right.
 
-## Final validation using Assurance
+## Validating Data Prep mapping using Assurance
 Now that the mapping is set up in the datastream, we have the full pathway of data:
 
 ```mermaid
 graph LR;
-    step1(App<br/>Analytics trackAction/trackState) --> step2(App<br/>Edge Bridge conversion to Edge event) --> step3(Edge Network<br/>Datastream translation of payload from contextdata to Edge XDM) --> step4(Edge Network<br/>Routing XDM data using datastream to Analytics);
+    step1(App<br/>Analytics track events) --> step2(App<br/>Edge Bridge conversion to Edge event) --> step3(Edge Network<br/>Datastream mapping of event data to XDM) --> step4(Edge Network<br/>Upstream applications);
 ```
 
-By using the Event Transactions view in the left-side navigation panel, the logical flow of events from `trackAction` event -> data mapping -> Analytics can be seen.
+By using the **Event Transactions** view in the left-side navigation panel, the logical flow of events from the event dispatched by the `trackAction` API -> data mapping can be seen.
 <img src="../assets/edge-bridge-tutorial/assurance-validation/assurance-event-transactions.png" alt="Assurance Event Transactions" width="1100"/>  
 
-Back in the Events view, the **mapper:xdm-event** (**1**) shows the result of the mapping from the Edge Bridge event's generic **data** -> **contextdata** structure into the XDM schema format (**2**).
+Back in the Events view, the **mapper:xdm-event** (**1**) shows the result of the mapping from the Edge Bridge event's data to the XDM schema format (**2**).
```

Comment:
could we show the mapper:xdm-event in the Event transactions view instead of navigating back to Events view?